nuitka/code_generation/CodeGeneration.py

```diff
@@ -896,6 +896,7 @@ def generateHelpersCode():
         "EXPRESSION_CTYPES_CDLL_REF": generateImportModuleNameHardCode,
         "EXPRESSION_CTYPES_CDLL_SINCE38_CALL": generateCtypesCdllCallCode,
         "EXPRESSION_CTYPES_CDLL_BEFORE38_CALL": generateCtypesCdllCallCode,
+        "EXPRESSION_CTYPES_C_INT_CALL": generateCtypesCIntCallCode,
```

Comment:
Usually there is also a REF node. But I think but hard imports these are added automatically. I never remember these things myself.